core/lexicon/ru/file.inc.php

```diff
@@ -61,7 +61,9 @@
 $_lang['file_folder_err_perms_remove'] = 'У веб-сервера не хватает прав для удаления каталога.';
 $_lang['file_folder_err_perms_upload'] = 'У веб-сервера не хватает прав для загрузки в каталог.';
 $_lang['file_folder_err_rename'] = 'Возникла неизвестная ошибка при переименовании каталога.';
+$_lang['file_folder_err_rename_protected'] = 'Переименование защищенного системного каталога не допускается.';
```

Comment:
All translations should be done via Crowdin, so, please, remove Russian strings from the pull request.